CONTRIBUTORS.txt

```diff
@@ -109,6 +111,7 @@ needs to be listed here.
 - Tomer Doron <tomerd@apple.com>
 - Trevör <adtrevor@users.noreply.github.com>
 - Trevör Anne Denise <adtrevor@users.noreply.github.com>
+- Volodymyr Sapsai <vsapsai@users.noreply.github.com>
```

Comment:
Hey @vsapsai , if you would like me to add a real email address here, please reply with the address you'd like me to use :)

Reply:
Current @users.noreply.github.com is good enough. Thanks for asking.